src/store/uiInteractions.js

```diff
@@ -13,6 +13,7 @@ export default {
     isResyncConfirmationVisible: false,
     isWalletDescriptionVisible: false,
     isWalletRenameVisible: false,
+    isDeleteWalletVisible: false,
```

Comment:
I found these variables a bit confusing. I think we should refactor them to something more understandable soon